sympy/functions/special/tests/test_singularity_functions.py

```diff
@@ -46,7 +46,7 @@ def test_eval():
     assert SingularityFunction(5, 6, -2) == 0
     assert SingularityFunction(4, 2, -2) == 0
     assert SingularityFunction(4, 4, -2) == oo
-    assert float(SingularityFunction(6.1, 4, 5)) == 40.84100999999997
+    assert (SingularityFunction(6.1, 4, 5)).evalf(5) == 40.84100
```

Comment:
Please read this: http://docs.sympy.org/dev/gotchas.html to see how sympy floats, python floats, and sympy evaluation work.